### PR TITLE
Test for restarting count

### PR DESCRIPTION
**Pull request by LK4D4**
This test is for #10058
ping @jfrazelle

### CODE REVIEW

**Comment by jfrazelle** [comment]
cool!

File: integration-cli/docker_cli_run_test.go [hunk 1]
```diff
@@ -2935,3 +2935,27 @@ func TestRunOOMExitCode(t *testing.T) {
...
+	logs := strings.TrimSpace(string(lout))
+	lines := strings.Split(logs, "\n")
+	for _, l := range lines {
+		if l != "docker" {
+			t.Fatalf("Log %q should contain only %q lines", logs, "docker")
+		}
+	}
+	if len(lines) != 4 {
+		t.Fatalf("Container was restarted %d times, expected %d", len(lines)-1, 3)
+	}
```
Comment (jfrazelle) [inline integration-cli/docker_cli_run_test.go]: do we need a log()

Comment (jfrazelle) [inline integration-cli/docker_cli_run_test.go]: also its failing locally :(

**Comment by icecrime** [comment]
I think this is could be made much simpler thanks to #9621:

```
$ docker inspect -f '{{.RestartCount}}' 3d4defe2061
3
```

**Comment by LK4D4** [comment]
@jfrazelle @icecrime PTAL

**Comment by icecrime** [comment]
LGTM

On Tuesday, January 13, 2015, Alexander Morozov <notifications@github.com>
wrote:

> @jfrazelle <https://github.com/jfrazelle> @icecrime
> <https://github.com/icecrime> PTAL
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/10075#issuecomment-69845769>.
>


-- 
Arnaud Porterie

**Comment by jfrazelle** [comment]
kewllll waiting on drone

**Comment by jfrazelle** [comment]
:cry:

**Comment by jfrazelle** [comment]
hmm it works locally

**Comment by LK4D4** [comment]
Haha, it's a race, I think I know solution

**Comment by LK4D4** [comment]
@icecrime @jfrazelle I think know there is no place for race

**Comment by icecrime** [comment]
Hehe nice :-) LGTM

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**